tests/integration/iop-integration-test-defaults.yaml

```diff
@@ -4,6 +4,13 @@ kind: IstioOperator
 metadata:
   name: install
 spec:
+  # TODO: make sure that this meshConfig is necessary
```

Comment:
Ok... so how do we make sure? :)

Reply:
As far as I remember, I left this comment when I was fixing multi-cluster deployment for federation. You can just remove this comment. I forgot to do it.

Reply:
92fff052d6